Source Packages/java/textdisplay/Project.java

```diff
@@ -1683,5 +1683,24 @@ public void setSchemaURL(Connection conn, String newURL) throws SQLException {
       }
    }
    
+   public String mintInterfaceLinkFromFolio(int folioNum) throws SQLException{
+        Folio f = new Folio(folioNum);
```

Comment:
We may want a possibility for a default number ([0] canvas, for example), but I'm not going to stop this.